src/main/java/pl/ug/virtualofficebackend/api/rest/UserController.java

```diff
@@ -0,0 +1,13 @@
+package pl.ug.virtualofficebackend.api.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+import pl.ug.virtualofficebackend.domain.user.boundary.UserService;
+
+@RestController
+public class UserController {
```

Comment:
Plus tutaj imo podział api/rest moze sie potem duzo zmienic jak bedziemy chcieli dodawac np. jakies eksporty
moze lepiej nazwac to np. api/Logging albo api/VirtualStudio czy jakos tak a potem dodawac kolejne pakiety razem z dodatkowymi funkcjanosciami

Reply:
Hmm, w sumie też spoko.
To może zmienić to w takim razie na api/rest/user, api/rest/logging, czy jakoś inaczej to myślisz aby zrobić?

Reply:
imo sam pakiet rest jest useless. Lepiej wg mnie grupowac to wg danej funkcjonalnosci. Np logging/export/virtualOffice/dashboards
w ten sposob bedzie wiadomo do ktorej funkcjonalnosci bedzie trzeba dodac dany endpoint

Reply:
logging/export/...? To w takim razie w logging byłyby wszystkie te funkcjonalności, do których trzeba się zalogować? To mi się raczej średnio podoba
Ale pakiet rest można wyrzucić, to wtedy możemy miec api/user, api/logging itd

Reply:
api/export
api/logging
api/virtualOffice
api/dashboards

Sorki troche zle to zapisalem xd

Reply:
Okej, to mi pasuje.
To w takim razie wszelakie DTO związane z tym będziemy wrzucać do konkretnego pakietu?

Reply:
Imo tak bedzie najczytelniej i potem bedzie latwo wprowadzic modyfikacje i nowe funkcjonalnosci